apps/web/modules/bookings/views/bookings-listing-view.tsx

```diff
@@ -73,11 +73,17 @@ const querySchema = z.object({
 
 export default function Bookings() {
   const params = useParamsWithFallback();
-  const { data: filterQuery } = useFilterQuery();
+  const { data: filterQuery, pushItemToKey } = useFilterQuery();
   const { status } = params ? querySchema.parse(params) : { status: "upcoming" as const };
   const { t } = useLocale();
   const user = useMeQuery().data;
   const [isFiltersVisible, setIsFiltersVisible] = useState<boolean>(false);
+  useEffect(() => {
+    if (user?.isTeamAdminOrOwner && !filterQuery.userIds?.length) {
+      setIsFiltersVisible(true);
+      pushItemToKey("userIds", user?.id);
+    }
+  }, [user, filterQuery.status]);
```

Comment:
`filterQuery.status` isn't used in the effect, so having it in the dependencies could trigger unnecessary re-runs. This version makes more sense:

```
useEffect(() => {
    if (user?.isTeamAdminOrOwner && (!filterQuery.userIds || !filterQuery.userIds.length)) {
        setIsFiltersVisible(true);
        pushItemToKey("userIds", user?.id);
    }
}, [user?.isTeamAdminOrOwner, filterQuery.userIds, user?.id]);

```

Reply:
`filterQuery.status` reflects the status tabs (upcoming, past, recurring,...),
On switching the tabs, the filters are reset.
So on tab change, this has to be re-run to set the default filter.